app/services/facebook.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Facebook < AuthServiceBase
```

Comment:
This class could be named `FacebookAuth`. A shame, however, that `AppleAuth` is already taken.

Reply:
We could just add Service to the end. So `AppleAuthService` and `FacebookAuthService`...